rpcserver.go

```diff
@@ -1068,7 +1069,8 @@ func allowCORS(handler http.Handler, origins []string) http.Handler {
 // address to a specified output value to be sent to that address.
 func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 	feeRate chainfee.SatPerKWeight, minConfs int32,
-	label string) (*chainhash.Hash, error) {
+	label string,
+	strategy wallet.CoinSelectionStrategy) (*chainhash.Hash, error) {
```

Comment:
nit: things can still fit on 3 lines instead of 4:

```
func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
	feeRate chainfee.SatPerKWeight, minConfs int32, label string,
	strategy wallet.CoinSelectionStrategy) (*chainhash.Hash, error) {
```